documentation/API_documentation/QoD_Latency_Bandwidth_User_Story.md

```diff
@@ -0,0 +1,10 @@
+# QoD Stable Bandwidth and Latency API User Story
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an application developer belonging to an enterprise, I want to request (using my application server/backend service), stable bandwidth/latency for a specified App-Flow from a Communication Service Provider (CSP), so that I can ensure better quality of experience for our end users during congestion.  |
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User<br> **Actors:** Application service providers, hyperscalers, application developers. Currently the API does not explicitly handle consent management (expects that the customer either owns the device subscriptions or has agreed to manage consent as a part of general Terms and Conditions) and hence end users are not included as Actors. As this changes, the actor list will be extended with end users. <br> **Scope:** *Order To Activate (OTA)* \- Create/Remove/Get service instance\, Get notification about the instance |
```

Comment:
Those Roles should be defined somewhere, carefully named so there is clear difference between them. (That might already be done somewhere and I missed it.)

Reply:
The roles are defined/suggested with https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/Deliverables/Userstory-template.md